modules/user_accounts/templates/form_edit_user.tpl

```diff
@@ -297,42 +297,40 @@ $(document).ready(function() {
     		{$form.CenterIDs.html}
     	</div>
     </div>
+    {if $form.errors.examiner_sites}
+    <div class="row form-group form-inline form-inline has-error">
+    {else}
+    <div class="row form-group form-inline form-inline">
+    {/if}
+        <label class="col-sm-2">
+            {$form.examiner_sites.label}
+        </label>
+        <div class="col-sm-10">
+            {$form.examiner_sites.html}
+        </div>
         {if $form.errors.examiner_sites}
-        <div class="row form-group form-inline form-inline has-error">
-            {else}
-            <div class="row form-group form-inline form-inline">
-                {/if}
-                <label class="col-sm-2">
-                    {$form.examiner_sites.label}
-                </label>
-                <div class="col-sm-10">
-                    {$form.examiner_sites.html}
-                </div>
-                {if $form.errors.examiner_sites}
-                    <div class="col-sm-offset-2 col-xs-12">
-                        <font class="form-error">{$form.errors.examiner_sites}</font>
-                    </div>
-                {/if}
+            <div class="col-sm-offset-2 col-xs-12">
+                <font class="form-error">{$form.errors.examiner_sites}</font>
             </div>
+        {/if}
+    </div>
+    {if $form.errors.examiner_group}
+    <div class="row form-group form-inline form-inline has-error">
```

Comment:
Same as above (although I do realize you just moved around an old piece of code)